spec/lib/companies_selector_spec.rb

```diff
@@ -0,0 +1,129 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'CompaniesSelector' do
```

Comment:
```suggestion
RSpec.describe CompaniesSelector do
```

Reply:
Ok, similar to controller or helpers testing form?

Reply:
you are writing a test for `CompaniesSelector` class and you can use that class as a description instead of using a string. I think it is a good practice to do that, but I don't have any confirmation from RSpec documentation, so you may want to search for it in the Internet if you want to.